src/njoy21/input/GROUPR/Card5/Sigz.hpp

```diff
@@ -0,0 +1,23 @@
+struct Sigz {
+  using Value_t = std::vector< Quantity< Barn > >;
+  static std::string name(){ return "sigz"; }
+
+  static std::string description() {
+    return
+        "The sigz argument is a list of sigma zero cross section values (in\n"
+        "barns) that are used to calculate resonance self shielding.";
+  }
+  static bool verify( const Value_t& sigz,
+                      const Argument< Card2::Nsigz > & nsigz ){
+    auto found = std::find_if( sigz.begin(), sigz.end(),
+                               [](auto& E){ return E < 0.0*barn; });
+
```

Comment:
Perhaps that's true, but we can't change the value because `sigz` is const, which I don't want to change.